src/azure-cli/azure/cli/command_modules/role/_help.py

```diff
@@ -9,31 +9,33 @@
 
 helps['ad'] = """
 type: group
-short-summary: Manage Azure Active Directory Graph entities needed for Role Based Access Control
+short-summary: >-
+    Manage Microsoft Entra ID (formerly known as Azure Active Directory, Azure AD, AAD) entities needed for
+    Azure role-based access control (Azure RBAC) through Microsoft Graph API.
 """
 
 helps['ad app'] = """
 type: group
-short-summary: Manage applications with AAD Graph.
+short-summary: Manage applications.
 """
 
 helps['ad app create'] = """
 type: command
-short-summary: Create a web application, web API or native application
+short-summary: Create an application.
 long-summary: For more detailed documentation, see https://docs.microsoft.com/graph/api/resources/application
 examples:
   - name: Create an application.
     text: |
         az ad app create --display-name mytestapp
-  - name: Create an application that can fall back to public client with Microsoft Graph delegated permission Application.Read.All
+  - name: Create an application that can fall back to public client with Microsoft Graph delegated permission User.Read
```

Comment:
`Application.Read.All` is a high-privileged permission requiring admin consent. The example should use a low-privileged permission.

Reply:
If `User.Read` is a permission, like `contributor`, can we put it in backticks so the sentence reads a bit clearer?